plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -2843,50 +2934,56 @@ bool producer_plugin_impl::push_read_only_transaction(transaction_metadata_ptr t
    auto retry = false;
 
    try {
-      auto start = fc::time_point::now();
+      auto               start = fc::time_point::now();
       chain::controller& chain = chain_plug->chain();
-      if ( !chain.is_building_block() ) {
-         _ro_exhausted_trx_queue.push_front( {std::move(trx), std::move(next)} );
+      if (!chain.is_building_block()) {
+         _ro_exhausted_trx_queue.push_front({std::move(trx), std::move(next)});
          return true;
       }
 
       // When executing a read-only trx on the main thread while in the write window,
       // need to switch db mode to read only.
-      auto db_read_only_mode_guard = fc::make_scoped_exit([&]{
-         if( chain.is_write_window() )
+      auto db_read_only_mode_guard = fc::make_scoped_exit([&] {
+         if (chain.is_write_window())
             chain.unset_db_read_only_mode();
       });
 
-      if ( chain.is_write_window() ) {
+      if (chain.is_write_window()) {
          chain.set_db_read_only_mode();
          auto idle_time = fc::time_point::now() - _idle_trx_time;
-         _time_tracker.add_idle_time( idle_time );
+         _time_tracker.add_idle_time(idle_time);
       }
 
       // use read-window/write-window deadline if there are read/write windows, otherwise use block_deadline if only the app thead
       auto window_deadline = (_ro_thread_pool_size != 0) ? _ro_window_deadline : _pending_block_deadline;
 
       // Ensure the trx to finish by the end of read-window or write-window or block_deadline depending on
-      auto trace = chain.push_transaction( trx, window_deadline, _ro_max_trx_time_us, 0, false, 0 );
+      auto trace = chain.push_transaction(trx, window_deadline, _ro_max_trx_time_us, 0, false, 0);
       _ro_all_threads_exec_time_us += (fc::time_point::now() - start).count();
-      auto pr = handle_push_result(trx, next, start, chain, trace, true /*return_failure_trace*/, true /*disable_subjective_enforcement*/, {} /*first_auth*/, 0 /*sub_bill*/, 0 /*prev_billed_cpu_time_us*/);
+      auto pr = handle_push_result(trx, next, start, chain, trace,
+                                   true /*return_failure_trace*/,
+                                   true /*disable_subjective_enforcement*/,
+                                   {} /*first_auth*/,
+                                   0 /*sub_bill*/,
+                                   0 /*prev_billed_cpu_time_us*/);
```

Comment:
Could use `//` comment instead of `/* */` now that these are on there own line.

Reply:
doing it!